crates/operator/src/lib.rs

```diff
@@ -52,7 +52,13 @@
 //! 2. **Create the operator configuration**: Create a [`OperatorConfig`](crate::config::OperatorConfig) struct.
 //!    This structs implements `Serialize` and `Deserialize` so you can load from a file.
 //!    - Attributes:
-//!      - `bls_private_key`: The BLS private key for
+//!      - `bls_signer_config`: The BLS signer for. We only support [web3-secret-storage](https://ethereum.org/es/developers/docs/data-structures-and-encoding/web3-secret-storage)
```

Comment:
This part is a bit weird

```suggestion
//!      - `bls_signer_config`: The config for the BLS signer. We only support [web3-secret-storage](https://ethereum.org/es/developers/docs/data-structures-and-encoding/web3-secret-storage)
```

Reply:
Done [1422d05](https://github.com/Layr-Labs/eigensdk-rs/pull/544/commits/1422d0599ef706bb0de650398a9e6f131eee9b54)